Sources/Amplitude/TrackingOptions.swift

```diff
@@ -160,7 +160,12 @@ public class TrackingOptions {
         return self
     }
 
+    @available(*, deprecated, message: "use static 'TrackingOptions.forCoppaControl()' instead")
```

Comment:
forCoppaControl is not a public function, right? Do we need the deprecation notice?

Reply:
We do not! I had thought that was weird, it turns out I had used a `@testable` import in my sample app which makes internal members public.